### ISSUE TITLE
Error message while starting the app

### ISSUE THREAD

**Issue by jakobroehrl** [stefan-niedermann/nextcloud-deck#807]
It only occurs 1x

```
Full Crash:

App Version: 1.14.0
App Version Code: 1014000
App Flavor: play

Files App Version Code: 30150051

---

OS Version: 4.14.117-perf+(2011052234)
OS API Level: 29
Device: OnePlus7T
Manufacturer: OnePlus
Model (and Product): HD1903 (OnePlus7T_EEA)

---

java.lang.NullPointerException: Attempt to invoke virtual method 'void it.niedermann.nextcloud.deck.ui.card.attachments.CardAttachmentAdapter.applyBrand(int)' on a null object reference
	at it.niedermann.nextcloud.deck.ui.card.attachments.CardAttachmentsFragment.applyBrand(CardAttachmentsFragment.java:518)
	at it.niedermann.nextcloud.deck.ui.branding.BrandedFragment.onStart(BrandedFragment.java:19)
	at androidx.fragment.app.Fragment.performStart(Fragment.java:2731)
	at androidx.fragment.app.FragmentStateManager.start(FragmentStateManager.java:365)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1206)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2637)
	at androidx.fragment.app.FragmentManager.dispatchStart(FragmentManager.java:2595)
	at androidx.fragment.app.FragmentController.dispatchStart(FragmentController.java:258)
	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:550)
	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:210)
	at it.niedermann.nextcloud.deck.ui.branding.BrandedActivity.onStart(BrandedActivity.java:22)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1514)
	at android.app.Activity.performStart(Activity.java:7843)
	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3412)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2123)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7710)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)

```

**Comment by stefan-niedermann** [comment]
Seems to be related (or even duplicate) to https://github.com/stefan-niedermann/nextcloud-deck/issues/782

I guess you got it only once and can not reproduce it?

**Comment by jakobroehrl** [comment]
> I guess you got it only once and can not reproduce it?

Right

**Comment by stefan-niedermann** [comment]
Yep, it's a duplicate of #782 - closing therefore and tracking it there :)

```
at it.niedermann.nextcloud.deck.ui.card.attachments.CardAttachmentsFragment.applyBrand(CardAttachmentsFragment.java:518)
at it.niedermann.nextcloud.deck.ui.branding.BrandedFragment.onStart(BrandedFragment.java:19)
```

**Comment by stefan-niedermann** [comment]
Will be fixed in `1.14.1`.
